src/components/box.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { AvailableSizes } from '../size';
+
+export interface IBoxProps {
+  size?: AvailableSizes;
```

Comment:
Nit (as seen in codebase / mentioned in reviews): should optional properties go last?